CONTRIBUTING.rst

```diff
@@ -0,0 +1,258 @@
+.. highlight:: shell
```

Comment:
It would be better to use `.. code-block:: shell` explicitly everywhere.

Remove the leading `$` as mentioned in other comment.